### PR TITLE
Mark some files as ignored; like yii2 repo

### PR DESCRIPTION
**Pull request by bscheshirwork**
Some values may be removed. I dont sure about this:

```
# composer itself is not needed
composer.phar

# phpunit itself is not needed
phpunit.phar
# local phpunit config
/phpunit.xml
```

Also, may be we need another files in gitignore
```

# composer.lock in applications is ignored since it's automatically created by composer when application is installed
/apps/*/composer.lock


# cubrid install dir
/cubrid

# ignore dev installed apps and extensions
/apps
/extensions
/packages

# NPM packages
/node_modules
.env

```

let's disscus!

| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  |

### CODE REVIEW

**Comment by samdark** [comment]
Merged. Thanks!

File: .gitignore [hunk 1]
```diff
@@ -1 +1,34 @@
...
+# visual studio code project files
+.vscode
+
+# windows thumbnail cache
+Thumbs.db
+
+# Mac DS_Store Files
+.DS_Store
+
+# composer vendor dir
```
Comment (cebe) [inline .gitignore]: hm.. this stuff should belong to a global gitignore file: 

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: .gitignore [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+# visual studio code project files
+.vscode
+
+# windows thumbnail cache
+Thumbs.db
+
+# Mac DS_Store Files
+.DS_Store
+
```
Comment (cebe) [inline .gitignore]: hm.. this stuff should belong to a global gitignore file: 

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (samdark) [reply .gitignore]: Ideally yes but we know that it may be not configured properly.

**Pull request merged**